packages/lumx-react/src/components/message/Message.stories.tsx

```diff
@@ -54,3 +54,17 @@ export const CustomIcon = {
         icon: mdiDelete,
     },
 };
+
+/**
+ * With close button (has background and kind info)
+ */
+export const ClosableMessage = {
+    args: {
+        closeButtonProps: {
+            onClose: () => console.log('test'),
+            label: 'Close',
+        },
+        hasBackground: true,
+        kind: 'info',
+    },
```

Comment:
Can we remove the console.log in favor of a SB action ?

Nested objects break when trying to combine `args` and `argTypes` but we have a `withNestedProps()` decorator that handles a custom `<prop>.<nestedprop>` notation to nest object 

```suggestion
    args: {
        'closeButtonProps.label': 'Close',
        hasBackground: true,
        kind: 'info',
    },
    argTypes: {
        'closeButtonProps.onClick':  { action: true },
    }
    decorators: [withNestedProps()]
```